db/fixtures/miq_user_roles.yml

```diff
@@ -14,7 +14,6 @@
   - embedded_automation_manager
   - availability_zone
   - host_aggregate
-  - compute
```

Comment:
Yea that seems confusing to me too.

There is no `compute` feature in `db/fixtures/miq_product_features.yml`.
So their reference in `miq_user_roles.yml` will just be a waisted lookup.

I used the following to determine if any invalid:

```
for i in `grep '^  -' db/fixtures/miq_user_roles.yml | sort -u`  ; do
  grep -q -- "${i}$" db/fixtures/miq_user_roles.yml || echo "MISSING: ${i}"
done
```